tool/gravity/cli/register.go

```diff
@@ -72,8 +72,8 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	_ = g.InstallCmd.Flag("docker-device", "[DEPRECATED] This flag will be removed in future version.").Hidden().String()
 	g.InstallCmd.SystemDevice = g.InstallCmd.Flag("system-device", "Device to use for system data directory.").Hidden().String()
 	g.InstallCmd.Mounts = configure.KeyValParam(g.InstallCmd.Flag("mount", "One or several mount overrides in the following format: <mount-name>:<path>, e.g. data:/var/lib/data."))
-	g.InstallCmd.PodCIDR = g.InstallCmd.Flag("pod-network-cidr", "Subnet range for Kubernetes pods network. Must be a minimum of /16.").Default(defaults.PodSubnet).String()
-	g.InstallCmd.ServiceCIDR = g.InstallCmd.Flag("service-cidr", "Subnet range for Kubernetes service networ.").Default(defaults.ServiceSubnet).String()
+	g.InstallCmd.PodCIDR = g.InstallCmd.Flag("pod-network-cidr", "[DEPRECATED] Use ClusterConfiguration resource to configure pod subnet.").Hidden().Default(defaults.PodSubnet).String()
```

Comment:
Nit: I think it's pretty convenient to be able to override things via CLI flags, so I'm not sure we should be deprecating them. Even if all we do is auto-generate a cluster configuration resource under the hoods when they are set.